notification/method/webpush.php

```diff
@@ -140,10 +146,18 @@ public function notify()
 	{
 		$insert_buffer = new \phpbb\db\sql_insert_buffer($this->db, $this->notification_webpush_table);
 
+		// Load all users we want to notify
+		$notify_users = $this->load_receivers();
```

Comment:
Why does this part need to be done here? I'm not understanding how this is related to what is happening in the notification loop below?

Reply:
This loads all users data which we need below to get `user_lang` for every `$notification->user_id`.

Reply:
It looks to me like all it does is set an array of user ids that aren't used until after these language swaps? 🤔

Reply:
It actually fills the data required in `$this->user_loader` object which we use several lines below. `$notify_users` array is needed just to be used in `notify_using_webpush()` later.